Cannot Build Project with make/cmake Easily

**Description**

I want to be able to run the project by running the `make` command in the root directory of this repo.
When I currently run `make`, I get the following output:

```bash

[ 25%] Building CXX object CMakeFiles/unofficial-discord-client.dir/main.o
[ 50%] Building CXX object CMakeFiles/unofficial-discord-client.dir/MainWindow.o
/home/bama4/REPO/unofficial-discord-client/src/MainWindow.cpp:4:10: fatal error: ui_MainWindow.h: No such file or directory
 #include "ui_MainWindow.h"
          ^~~~~~~~~~~~~~~~~
compilation terminated.
CMakeFiles/unofficial-discord-client.dir/build.make:86: recipe for target 'CMakeFiles/unofficial-discord-client.dir/MainWindow.o' failed
make[2]: *** [CMakeFiles/unofficial-discord-client.dir/MainWindow.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/unofficial-discord-client.dir/all' failed
make[1]: *** [CMakeFiles/unofficial-discord-client.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

I had to look inside of `.github/workflows/build.yml` to be able to know how to build the project.

**Goal**
- [x] Be able to build by running cmake
- [x] Add instructions on building the project to the README.md file

--- a/README.md
+++ b/README.md
@@ -2,4 +2,19 @@
 
 ![Build](https://github.com/Coding-Bunker/unofficial-discord-client/workflows/Build/badge.svg)
 
+# Description
+
 The main purpose of this client is to be a desktop application fast and simple to use in order to replace the official application. Is a community project and anyone can help with issues, feedback or coding. The purpose is to learn how manage desktop application in C++ with Qt framework using public API provided by discord.
+
+
+## Build
+
+- Qt min version 5.15.1
+- CMake min version 3.5
+
+```
+mkdir build
+cd build
+cmake ..
+cmake --build .
+```

